x/ecocredit/base/keeper/features/msg_create_or_update_application.feature

```diff
@@ -61,4 +61,11 @@ Feature: Msg/CreateOrUpdateApplication
       Then expect error contains "unauthorized"
       And expect the application for "P001" to "C01" exists with metadata "abc123"
 
-  Rule: events get emitted
+  Rule: project admins cannot withdraw a project with an accepted enrollment (a credit class issuer must do that)
```

Comment:
```suggestion
  Rule: project admins cannot withdraw a project with an accepted enrollment
```

Just got hung up on this getting my head back into things.. this implies a credit class issuer can withdraw a project. But only project admins can attempt a withdraw via `MsgCreateorUpdateApplication`. To remove the project enrollment from state credit class issuers must change enrollment status from `ACCEPTED` -> `TERMINATED`. Maybe just drop `(a credit class issuer must do that)` for clarity?